sdk/identity/identity/rollup.base.config.js

```diff
@@ -88,7 +87,7 @@ export function browserConfig(test = false) {
   };
 
   if (test) {
-    baseConfig.input = "dist-esm/test/*.spec.js";
+    baseConfig.input = "dist-esm/test/internal/*.spec.js";
```

Comment:
Why is public not included here?

Reply:
My bad, I should have used this approach: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/rollup.base.config.js#L59 I'll make a new PR.

Reply:
Why this was only internal before is because many of the public tests are only supposed to work in Node. I will figure a way around this as part of [this PR](https://github.com/Azure/azure-sdk-for-js/pull/10968/files#r481413974).